code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -497,6 +497,33 @@
 		visible_message("<span class='notice'>[src] fizzes ominously.</span>")
 		addtimer(CALLBACK(src, .proc/fizzbuzz), 110)
 
+//lazy citadel edit begin
+/obj/machinery/nuclearbomb/beer/aphro
+	name = "Nanotrasen-brand nuclear fission explosive"
+	desc = "One of the more successful achievements of the Nanotrasen Corporate Warfare Division, their nuclear fission explosives are renowned for being cheap to produce and devastatingly effective. Signs explain that though this particular device has been decommissioned, every Nanotrasen station is equipped with an equivalent one, just in case. All Captains carefully guard the disk needed to detonate them - at least, the sign says they do. There seems to be a tap on the back. It smells of a sickening sweet flavor."
+
+/obj/machinery/nuclearbomb/beer/aphro/Initialize()
+	. = ..()
+	if(keg)
+		keg.reagents.remove_any(INFINITY)
```

Comment:
this is not how you do things, you want to go add support for a starting reagent to the /obj/machinery/nuclearbomb/beer path, bro you know this by now

Reply:
>not even on it's own modular_cit